src/Icicle/Storage/Dictionary/Toml.hs

```diff
@@ -150,10 +153,7 @@ checkDefs d defs
  where
   go (a, q)
    = do  (checked, _)  <- check' d q
-         let inlined    = P.sourceInline d checked
-         blanded       <- hoistEither . mapLeft DictionaryErrorTransform $ P.sourceDesugarQT inlined
-         (checked', _) <- check' d blanded
-         pure $ DictionaryEntry a (VirtualDefinition (Virtual checked'))
+         pure $ DictionaryEntry a (VirtualDefinition (Virtual checked))
```

Comment:
but it's also like - we might as well do the whole "compilation" in the one place. so it's either have this do the conversion to core and avalanche, or do it all later